shell/app/common/components/tree/file-tree.scss

```diff
@@ -0,0 +1,54 @@
+.file-tree-container {
+  .tree-node-action {
+    position: absolute;
+    right: 5px;
+    top: calc(50% - 8px);
+    display: none;
+  }
+
+  .ant-tree-treenode:hover .tree-node-action {
+    display: inline-block;
+  }
+
+  .ant-tree-treenode:before {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    height: 40px;
+    transition: all 0.3s;
+  }
+
+  .ant-tree-treenode-selected:before,
+  .ant-tree-treenode:hover:before {
+    background: #f6f5f9;
```

Comment:
Where is this color from?

Reply:
This is the color of the NUSI component.